package/base-files/files/lib/preinit/80_mount_root

```diff
@@ -46,6 +46,10 @@ do_mount_root() {
 		missing_lines /tmp/group /etc/group >> /etc/group
 		missing_lines /tmp/shadow /etc/shadow >> /etc/shadow
 		rm /tmp/passwd /tmp/group /tmp/shadow
+
+		if [ "$(ls -A /etc/uci-defaults/ | wc -l)" -eq 0 ]; then
```

Comment:
is the guarding if clause really needed here ?

Reply:
Should I remove guarding if and request new Pull request?

Reply:
just update you patch and force push it. do not reopen the PR, just fix this one

Reply:
done~